test/tasks/demo.test.js

```diff
@@ -162,5 +162,27 @@ describe('Demo task', function() {
 
 			demoStream.resume();
 		});
+
+		it('should load local partials', function(done) {
+			fs.mkdirSync('demos/src/partials');
+			fs.writeFileSync('demos/src/test1.mustache', '<div>test1</div>{{>partial1}}', 'utf8');
+			fs.writeFileSync('demos/src/partial1.mustache', '<div>partial1</div>', 'utf8');
+			fs.writeFileSync('demos/src/test2.mustache', '<div>test1</div>{{>partials/partial2}}', 'utf8');
+			fs.writeFileSync('demos/src/partials/partial2.mustache', '<div>partial2</div>', 'utf8');
```

Comment:
Might be worth doing then only for the files that aren't used in the other tests?